cmd/opentelemetry-collector/app/exporter/memory/factory_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package memory
+
+import (
+	"context"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configcheck"
+	"go.opentelemetry.io/collector/config/configerror"
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.uber.org/zap"
+
+	jConfig "github.com/jaegertracing/jaeger/pkg/config"
+)
+
+func TestCreateTraceExporter(t *testing.T) {
+	v, _ := jConfig.Viperize(AddFlags)
+	factory := &Factory{Viper: v}
+	exporter, err := factory.CreateTraceExporter(context.Background(), component.ExporterCreateParams{Logger: zap.NewNop()}, factory.CreateDefaultConfig())
+	require.NoError(t, err)
+	require.NotNil(t, exporter)
+}
```

Comment:
Should there be a test to ensure that the exporter is using a singleton?

Reply:
+1 added